/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py:335: RuntimeWarning: invalid value encountered in subtract
  lnpdiff = (self.dim - 1.) * np.log(zz) + newlnprob - lnprob0
/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py:336: RuntimeWarning: invalid value encountered in greater
  accept = (lnpdiff > np.log(self._random.rand(len(lnpdiff))))
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
lf_fitter.py:80: RuntimeWarning: divide by zero encountered in log10
  phi_i = np.interp(L_data, L_model, np.log10(PHI_model))
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in divide
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: overflow encountered in power
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:11: RuntimeWarning: divide by zero encountered in log10
  P_temp = np.log10(P_temp)
/home/xuejian/works/quasarLF/git/codes/lf_fit/lf_shape.py:10: RuntimeWarning: divide by zero encountered in divide
  P_temp = np.power(10.,P0) / ( np.power(x,gamma1) + np.power(x,gamma2) )
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newx_load_aird15_lf_data.py", line 11, in load_aird15_lf_data_softsel
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1713, in genfromtxt
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/codecs.py", line 314, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newx_load_aird15_lf_data.py", line 11, in load_aird15_lf_data_softsel
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1713, in genfromtxt
    first_line = _decode_line(next(fhd), encoding)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/codecs.py", line 314, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
MemoryError
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/new_load_palanque13_lf_data.py", line 12, in load_palanque13_lf_data
    'left', 'right', 'log_phi', 'uperr', 'lowerr', 'nqso', 'Veff', 'P'])
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/new_load_palanque13_lf_data.py", line 12, in load_palanque13_lf_data
    'left', 'right', 'log_phi', 'uperr', 'lowerr', 'nqso', 'Veff', 'P'])
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newir_load_lacy15_lf_data.py", line 11, in load_lacy15_lf_data
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newir_load_lacy15_lf_data.py", line 11, in load_lacy15_lf_data
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
    encoding=encoding, newline=newline)
MemoryError
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newir_load_lacy15_lf_data.py", line 11, in load_lacy15_lf_data
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newir_load_lacy15_lf_data.py", line 11, in load_lacy15_lf_data
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
MemoryError
Traceback (most recent call last):
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/new_load_palanque13_lf_data.py", line 12, in load_palanque13_lf_data
    'left', 'right', 'log_phi', 'uperr', 'lowerr', 'nqso', 'Veff', 'P'])
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/new_load_willott10_lf_data.py", line 11, in load_willott10_lf_data
    'left', 'right', 'log_phi', 'uperr', 'lowerr', 'nqso', 'Veff', 'P'])
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/new_load_willott10_lf_data.py", line 11, in load_willott10_lf_data
    'left', 'right', 'log_phi', 'uperr', 'lowerr', 'nqso', 'Veff', 'P'])
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/new_load_palanque13_lf_data.py", line 12, in load_palanque13_lf_data
    'left', 'right', 'log_phi', 'uperr', 'lowerr', 'nqso', 'Veff', 'P'])
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
Traceback (most recent call last):
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newx_load_khorunzhev18_lf_data.py", line 11, in load_khorunzhev18_lf_data_vitobins
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/new_load_willott10_lf_data.py", line 11, in load_willott10_lf_data
    'left', 'right', 'log_phi', 'uperr', 'lowerr', 'nqso', 'Veff', 'P'])
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newx_load_khorunzhev18_lf_data.py", line 11, in load_khorunzhev18_lf_data_vitobins
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/new_load_willott10_lf_data.py", line 11, in load_willott10_lf_data
    'left', 'right', 'log_phi', 'uperr', 'lowerr', 'nqso', 'Veff', 'P'])
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newir_load_lacy15_lf_data.py", line 11, in load_lacy15_lf_data
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newir_load_lacy15_lf_data.py", line 11, in load_lacy15_lf_data
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/load_sdss_dr3_lf_data.py", line 23, in load_sdss_dr3_lf_data
    data = np.genfromtxt(filename,names=['M_i','logphi','dphi','fill','zbar','Nqso','Ncor'],skip_header=1)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newx_load_aird15_lf_data.py", line 11, in load_aird15_lf_data_softsel
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1713, in genfromtxt
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/codecs.py", line 314, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
MemoryError
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/load_sdss_dr3_lf_data.py", line 23, in load_sdss_dr3_lf_data
    data = np.genfromtxt(filename,names=['M_i','logphi','dphi','fill','zbar','Nqso','Ncor'],skip_header=1)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
Traceback (most recent call last):
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/newx_load_aird15_lf_data.py", line 11, in load_aird15_lf_data_softsel
    data = np.genfromtxt(filename,names=True)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1713, in genfromtxt
    first_line = _decode_line(next(fhd), encoding)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/codecs.py", line 314, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/load_sdss_dr3_lf_data.py", line 23, in load_sdss_dr3_lf_data
    data = np.genfromtxt(filename,names=['M_i','logphi','dphi','fill','zbar','Nqso','Ncor'],skip_header=1)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
MemoryError
Traceback (most recent call last):
  File "lf_fitter.py", line 163, in <module>
    pool.wait()
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/mpi_pool.py", line 117, in wait
    result = self.function(task)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/emcee/ensemble.py", line 519, in __call__
    return self.f(x, *self.args, **self.kwargs)
  File "lf_fitter.py", line 141, in lnprob
    llike = lnlike(pars)
  File "lf_fitter.py", line 120, in lnlike
    res, sigma, success = residual(pars)
  File "lf_fitter.py", line 105, in residual
    get_fit_data(alldata,parameters,zmins[key],zmaxs[key],key,dset_ids[key])
  File "lf_fitter.py", line 43, in get_fit_data
    L_data, PHI_data, DPHI_data = load_LF_data[dset_name](redshift)
  File "/home/xuejian/works/quasarLF/git/codes/obdata/load_sdss_dr3_lf_data.py", line 23, in load_sdss_dr3_lf_data
    data = np.genfromtxt(filename,names=['M_i','logphi','dphi','fill','zbar','Nqso','Ncor'],skip_header=1)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/npyio.py", line 1689, in genfromtxt
    fhd = iter(np.lib._datasource.open(fname, 'rt', encoding=encoding))
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 260, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/home/xuejian/.conda/envs/customenv/lib/python2.7/site-packages/numpy/lib/_datasource.py", line 614, in open
    encoding=encoding, newline=newline)
MemoryError
slurmstepd: error: *** JOB 203717 ON wheeler088 CANCELLED AT 2019-11-01T11:38:09 DUE TO TIME LIMIT ***
